ofl/bizudmincho/METADATA.pb

```diff
@@ -20,5 +20,220 @@ subsets: "latin-ext"
 subsets: "menu"
```

Comment:
I don't see these subsets:
subsets: "chinese-simplified"
subsets: "chinese-traditional"
@aaronbell is it normal?